akvo/rsr/models/result.py

```diff
@@ -146,7 +146,7 @@ def child_projects(self):
         Return a dictionary of this result's child projects.
         """
         projects = {}
-        for result in Result.objects.filter(parent_result=self):
+        for result in Result.objects.filter(parent_result=self).select_related('project__title'):
```

Comment:
A nitpick: `select_related` is a way of asking related objects to be fetched in the same query, instead of making queries in the future. `select_related('project__title')` is therefore exactly equivalent to `select_related('project')` (verified by comparing the actual queries).  Can we drop the `__title`?